spec/features/proposals_spec.rb

```diff
@@ -203,6 +203,89 @@
         expect(page).not_to have_link("No comments", href: "#comments")
       end
     end
+
+    scenario "Shows clickable markers for related proposals by tag on proposal map", :js do
+      Setting["feature.map"] = true
+      create(:tag, :category, name: "culture")
+      create(:tag, :category, name: "sports")
+      proposal = create(:proposal, tag_list: "culture, sports", map_location: create(:map_location))
+      related_proposal = create(:proposal, tag_list: "culture", map_location: create(:map_location, latitude: 51.50))
+      create(:proposal, tag_list: "sports")
```

Comment:
I was wondering why this proposal wasn't showing on the map. I know I should have noticed it, but since I didn't :sweat_smile:, how about adding `map_location: nil` to make it more explicit?